src/main.rs

```diff
@@ -174,6 +174,33 @@ fn main() -> Result<()> {
             let template_name = matches.value_of("KEY").unwrap();
             let body = templates.get_body_by_name(org_id, env, template_name)?;
 
+            if let Some(body) = body {
+                println!("{}", body)
+            } else {
+                println!(
+                    "Could not find a template with name '{}' in environment '{}'.",
+                    template_name,
+                    env.unwrap_or("default")
+                )
+            }
+        } else if let Some(matches) = matches.subcommand_matches("getit") {
+            let starts_with = matches.value_of("starts_with");
+            let ends_with = matches.value_of("ends_with");
+            let contains = matches.value_of("contains");
+            let template_name = matches.value_of("KEY").unwrap();
```

Comment:
If you do rename the argument to `"NAME"`, make sure you update it here as well.